tenants/infra.yaml

```diff
@@ -66,6 +67,11 @@ spec:
         url: "oci://ghcr.io/controlplaneio-fluxcd/d2-infra/<< inputs.tenant >>"
         ref:
           tag: << inputs.tag >>
+        verify:
+          provider: cosign
+          matchOIDCIdentity:
+          - issuer: ^https://token\.actions\.githubusercontent\.com$
+            subject: ^https://github\.com/controlplaneio-fluxcd/d2-infra/\.github/workflows/push-artifact\.yaml@<< inputs.ociSubjectGitRef >>$
```

Comment:
This would not work, there is a different workflow for production artifacts.

Reply:
Fixed, thanks!